jobs/build/advisories/advisories.groovy

```diff
@@ -67,4 +67,32 @@ def create_placeholder(kind) {
     echo "out: ${out}"
 }
 
+def create_textonly() {
+  cmd = [
+      "--group openshift-${params.VERSION}",
+      "create-textonly",
+      "--synopsis ${params.SYNOPIS}",
+      "--topic ${params.TOPIC}",
+      "--description ${params.DESCRIPTION}",
+      "--solution ${params.SOLUTION}",
+      "--bugtitle ${params.BUGTITLE}",
+      "--bugdescription ${params.BUGDESCRIPTION}",
+      "--assigned-to ${params.ASSIGNED_TO}",
+      "--manager ${params.MANAGER}",
+      "--package-owner ${params.PACKAGE_OWNER}"
+  ].join(' ')
+  if (params.DATE != null) {
```

Comment:
I think `params.DATE` will be empty string instead of `null` if the parameter is blank.
```suggestion
  if (params.DATE) {
```